lib/src/parse/parse.dart

```diff
@@ -431,32 +427,15 @@ void parse<D>(
 
     if (elementSpec.modifiers != null) {
       for (var modifier in elementSpec.modifiers!) {
-        GeomModifierOp geomModifier;
-        if (modifier is DodgeModifier) {
```

Comment:
Most important change: we no longer need to check the instance type of `modifier`. Every modifier must implement a `toGeomModifierOp` that returns the corresponding `GeomModifierOp`, which allows everyone to specify their custom modifiers.